erts/emulator/nifs/win32/win_prim_file.c

```diff
@@ -491,6 +491,42 @@ posix_errno_t efile_open(const efile_path_t *path, enum efile_modes_t modes,
     }
 }
 
+posix_errno_t efile_from_fd(int fd,
+                            enum efile_modes_t modes,
+                            ErlNifResourceType *nif_type,
+                            efile_data_t **d) {
+    HANDLE handle;
+
+    handle = (HANDLE)_get_osfhandle(fd);
+
+    if(handle != INVALID_HANDLE_VALUE) {
```

Comment:
You should check for `-2` as well.

https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/get-osfhandle?view=msvc-160

Reply:
Thanks, I should have noticed that text in the note.

Reply:
Fixed: 62a29aa